releasenotes/releasenotes.rst

```diff
@@ -24,6 +24,55 @@ Public Endpoint Announcement 8 July 2021
 
 The default FHIR version of the `public Firely Server endpoint <https://server.fire.ly/>`_ is now R4.
 
+.. _vonk_releasenotes_470:
+
+Release 4.7.0, Feb 1st, 20222
+-----------------------------
+
+.. attention::    
+    With version 4.7.0, Firely Server migrated to .NET 6.0. In order to run the binaries, `ASP.NET Core Runtime 6.x <https://dotnet.microsoft.com/en-us/download/dotnet/6.0>`_ needs to be installed.
+
+
+Feature
+^^^^^^^
+
+#. BulkDataExport is now supported for MongoDB as well. Get started with the :ref:`Bulk Data Export documentation<feature_bulkdataexport>`.
```

Comment:
Did you check all the existing BDE docs to see if something is said about MongoDB support? E.g. the [BDE page](https://docs.fire.ly/projects/Firely-Server/en/latest/features/bulkdataexport.html) has a note saying that we don't support it.

Reply:
Also check all related settings documentation.

Reply:
Will address this in a separate PR, see VONK-3137